### DISCUSSION TITLE
Install and then update

### DISCUSSION THREAD

**Discussion by t794104** [chocolatey/choco#2439] [Q&A] [answered]
If created a choco package and use Get-PackageParameters to check if we want a alternate install location. And this works fine; defautl is 'c:\program file\blabla' and in can overwrite it during install

But when i release a new version of that package and do a 'choco upgrade all' it removes the old version and install the new in it's default location not in it's alternat location.

How can i make it that a upgrade uses the same packageParams at the orginal install ?

**Comment by corbob** [comment, marked as answer]
Sounds like you want to enable `useRememberedArgumentsForUpgrades` - Use Remembered Arguments For Upgrades - when running upgrades, use arguments for upgrade that were used for installation ('remembered'). This is helpful when running upgrade for all packages. Available in 0.10.4+. This is considered in preview for 0.10.4 and will be flipped to on by default in a future release.
https://docs.chocolatey.org/en-us/configuration

IIRC the command to enable it would be `choco feature enable -n useRememberedArgumentsForUpgrades`
